src/azure-cli/azure/cli/command_modules/dms/custom.py

```diff
@@ -156,11 +158,13 @@ def create_task(cmd,
                 source_connection_json,
                 target_connection_json,
                 database_options_json,
+                optional_agent_settings,
```

Comment:
Instead of adding a new parameter, add the contents of optional_agent_settings as part of database_options_json.

Reply:
done